tests/h/services/search_index/_queue_test.py

```diff
@@ -120,6 +140,29 @@ def test_it_ignores_jobs_beyond_limit(
         for annotation_id in all_annotation_ids[LIMIT:]:
             assert annotation_id not in batch_indexer.index.call_args[0][0]
 
+    def test_if_the_job_has_force_True_it_indexes_the_annotation_and_deletes_the_job(
+        self,
+        annotations,
+        annotation_ids,
+        batch_indexer,
+        caplog,
+        db_session,
+        index,
+        queue,
+    ):
+        index(annotations)
+        queue.add_all(
+            annotation_ids, tag="test_tag", schedule_in=MINUS_FIVE_MINUTES, force=True
+        )
+
+        queue.sync(LIMIT)
+
+        assert str({FORCED: 10}) in caplog.text
+        assert db_session.query(Job).all() == []
+        batch_indexer.index.assert_called_once_with(
+            Any.list.containing(annotation_ids).only()
+        )
```

Comment:
Do we want to add any testing here about the fact we don't pull these ones back from ES and Postgres for comparison?

Reply:
I think I'm gonna pass on this one, as I can't think of a good way to test it. Given that the tests use the real Postgres and Elastic, how can you test that it _doesn't_ query them? It's not that it doesn't query the DB at all either, since it does query it to get the jobs, it's that it doesn't do the annotations query. I could replace Postgres and Elastic with mocks for one test but those APIs are super ugly to mock. I think you might have to do it by patching the `_get_annotations_from_db()` and `_get_annotations_from_es()` methods but that'd couple the tests to the implementation. I can't think of a good way